albumentations/augmentations/domain_adaptation.py

```diff
@@ -24,6 +29,16 @@ def fourier_domain_adaptation(img: np.ndarray, target_img: np.ndarray, beta: flo
         transformed image
 
     """
+
+    img = np.squeeze(img)
+    target_img = np.squeeze(target_img)
+
+    if target_img.shape != img.shape:
+        raise ValueError(
+            "The source and target images must contain the same shape,"
```

Comment:
A small improvement of the exception message "The source and target images must _have_ the same shape,"

Reply:
done